mobile/src/bundles/employer/constants/constants.ts

```diff
@@ -0,0 +1,126 @@
+//TODO delete mock-data
+import { type ChatData } from '~/bundles/employer/types/types';
+
+const imageAvatar1 =
```

Comment:
not critical for mock, but we dont use such naming

`firstImageAvatar` - looks better